packages/react-wildcat-handoff/src/utils/serverRender.js

```diff
@@ -1,10 +1,8 @@
-"use strict";
-
 const ReactDOM = require("react-dom/server");
 const Router = require("react-router");
 const serverContext = require("./serverContext.js");
 
-const Helmet = require("react-helmet");
+const Helmet = require("react-helmet"); // eslint-disable-line import/no-unresolved
```

Comment:
Neither npm nor Yarn dedupe linked packages. It's not usually a big deal, but in this case Helmet needs to share the same singleton in both `react-wildcat-handoff` and `example`, in order to test and confirm the correct server-side output.

So when you run `make install`, I added logic to remove the duped `react-helmet` instance and [replace it with a symlink](https://github.com/nfl/react-wildcat/pull/194/files#diff-70d02df1f67672b7a5a7d231d92a8a46). This allows me to test the correct output, but has the side-effect of ESLint not detecting `react-helmet` because it does not follow symlinks. So the workaround here was to disable import validation.